src/routes/(infoPages)/[topTierPage]/[...serviceGroupPage]/+page.server.ts

```diff
@@ -305,7 +313,7 @@ export const load = async ({
     if (!pageMetadata) break fetchData;
     const client = getContentfulClient({
       spaceID: CONTENTFUL_SPACE_ID,
-      token: CONTENTFUL_DELIVERY_API_TOKEN,
+      token: CONTENTFUL_PREVIEW_API_TOKEN,
```

Comment:
Added this in to work on developing the feature/displaying it on the test site. will remove this and the `preview:true` flags before merging